### DISCUSSION TITLE
SOLWEIG error message "Error: No valid DEM selected"

### DISCUSSION THREAD

**Discussion by Witellsteim** [UMEP-dev/UMEP#526] [Q&A] [answered]
Hello all,

I receive the following message when trying to run SOLWEIG with minimal input parameters from the Tutorial Testdata:
![Screenshot 2023-06-06 141128](https://github.com/UMEP-dev/UMEP/assets/124663247/c1e1e0ae-bc8f-4298-a9d9-2c8adc2b6d68)

The module runs succesful however if the optional DEM input is set. 
![Screenshot 2023-06-06 141308](https://github.com/UMEP-dev/UMEP/assets/124663247/eddf0466-c8ca-4825-b460-bf80dd729010)

I was wondering if this is a naming error, since it seems to be mandatory and not optional or whether i did something obscure without realizing it?

Cheers and best regards,
W.

**Comment by biglimp** [comment, marked as answer]
SOLWEIG needs to know the location of buildings to be able to calculate radiation fluxes correctly. Therefore, the model requires either and land cover dataset OR a DEM tolocate buildings [(DSM - DEM) > 0 = buildings]. If none of them are loaded, the error that you experience is shown. We will clarify with a more informative error message.
